README.md

```diff
@@ -3,47 +3,58 @@
   - SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
-# Vue components
+# @nextcloud/vue
 
 [![REUSE status](https://api.reuse.software/badge/github.com/nextcloud-libraries/nextcloud-vue)](https://api.reuse.software/info/github.com/nextcloud-libraries/nextcloud-vue)
-[![npm last version](https://img.shields.io/npm/v/@nextcloud/vue.svg?style=flat-square)](https://www.npmjs.com/package/@nextcloud/vue)
-[![Dependabot status](https://img.shields.io/badge/Dependabot-enabled-brightgreen.svg?longCache=true&style=flat-square&logo=dependabot)](https://dependabot.com)
+[![npm last version](https://img.shields.io/npm/v/@nextcloud/vue.svg?style=flat)](https://www.npmjs.com/package/@nextcloud/vue)
+[![Dependabot status](https://img.shields.io/badge/Dependabot-enabled-brightgreen.svg?longCache=true&style=flat&logo=dependabot)](https://dependabot.com)
 
-This repo contains the various Vue.js components that Nextcloud uses for its internal design and structure. It provides standardized UI elements for building Nextcloud app frontends with Vue.js.
+> 🖼️ Nextcloud UI Kit for building Nextcloud apps with Vue
 
-## Documentation
+- ✨ Standardized UI Components
+- 🛠️ Composables and frontend utilities
+- 🔗 [Reference providers](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/reference.html) utilities
 
-A list of available components with examples to try out is available in the [documentation](https://nextcloud-vue-components.netlify.app).
+## 📄 Documentation
 
-The documentation is built from the latest development branch, for stable releases the documentation can be found matching the latest minor version:
+| Version        | Target                | Documentation                                         |
+|----------------|-----------------------|-------------------------------------------------------|
+| v9.x [next]    | Nextcloud 28+ (Vue 3) | https://next--nextcloud-vue-components.netlify.app    |
```

Comment:
```suggestion
| v9.x [next]    | Nextcloud 29+ (Vue 3) | https://next--nextcloud-vue-components.netlify.app    |
```

We should not promote support for unsupported Nextcloud versions in a new release, this could bit us later.
Maybe even state:

```suggestion
| v9.x [next]    | Nextcloud 30+ (Vue 3) | https://next--nextcloud-vue-components.netlify.app    |
```

Reply:
Yes but my reasoning are the design changes in 30. Also 29 will be EOL shortly after 32 release and I do not expect us to ne quicker. 

But yes we can change that later